gimme

```diff
@@ -18,6 +18,8 @@
 #+        -f --force force - remove the existing go installation if present prior to install
 #+          -l --list list - list installed go versions and exit
 #+        -k --known known - list known go versions and exit
+#+    --force-known-update - when used with --known, ignores the cache and updates
+#+    -r --resolve resolve - resolve a version specifier to a version, show that and exit
```

Comment:
I think it is worth mentioning here that version _must_ be given as an argument. `GIMME_GO_VERSION` will not work.

Reply:
oops, good call; I changed the code to handle `GIMME_GO_VERSION` instead though